docs/menu.yml

```diff
@@ -189,6 +189,8 @@ catalog:
             catalog:
               - name: "EKS Monitoring"
                 path: "/en/setup/backend/backend-aws-eks-monitoring"
+              - name: "S3 Monitoring"
+                path: "/en/setup/backend/backend-aws-s3-monitoring"
```

Comment:
Just FYI, we plan to add DynamoDB on both AWS and Database menu(UI and docs)

https://github.com/apache/skywalking/pull/10380#discussion_r1103720431